typescript/packages/core/src/plugins/solana/nfts.ts

```diff
@@ -0,0 +1,51 @@
+import { PublicKey } from "@solana/web3.js";
+import type { SolanaWalletClient } from "../../wallets";
+import type { Plugin } from "../plugins";
+import { z } from "zod";
+import { mplBubblegum, getAssetWithProof, transfer } from "@metaplex-foundation/mpl-bubblegum";
+import { createUmi } from "@metaplex-foundation/umi-bundle-defaults";
+import { fromWeb3JsPublicKey, toWeb3JsInstruction } from "@metaplex-foundation/umi-web3js-adapters";
+
+export function nfts(): Plugin<SolanaWalletClient> {
+    return {
+        name: "nft_actions",
+        supportsSmartWallets: () => true,
```

Comment:
I think this is not correct (yet)

Reply:
It will be when smart wallets exist without changes though. I think the real correct course of action would be to remove this from the interface since not all chains will have smart wallets @0xaguspunk

Reply:
We're actually not sure yet if solana smart wallets will support this without changes, because of the ongoing CPI limit conversation which can affect cNFTs of certain tree sizes. If there's no downside I would mark it as false, and only mark it true once smart wallet support has been released in Solana and it's been tested!

Reply:
Ok fair enough